Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -46,7 +59,184 @@ need to be enabled.
+
+*   All active rules can be seen in the backend configuration section, see
+    :ref:`content-security-policy-backend-rules`.
+
+..  _content-security-policy-example:
+
+Example scenario
+================
+
+Let's define a small real-world scenario:
+
+*   You have one TYPO3 installation with two sites (frontend), `example.com` and `example.org`.
+*   You have created custom backend modules for some distinct functionality.
+*   `example.com` is a site where your editors fully control all frontend content, and they
+    want to have full flexibility of what and how to embed. You use a CDN network to deliver
+    your own large assets, like videos.
+*   `example.org` is a community-driven site, where users can manage profiles and post chats,
+    and where you want to prevent exploits on your site. Some embedding to a set of allowed
+    web services (YouTube, Google Analytics) must be possible.
+
+So you need to take care of security measures, and find a pragmatic way how to allow foreign
+content (like YouTube, widgets, tracking codes, assets)
+
+Specifically you want to to set these rules, as an example:
+
+Rules for example.com (editorial)
+---------------------------------
+
+*    :html:`<iframe>` to anyhwere should be allowed
+*    :html:`<img>` sources to anywhere should be allowed
+*    :html:`<script>` sources to `cdn.example.com` and `*.youtube.com` and `*.google.com`
+     should be allowed
+
+Rules for example.org (community)
+---------------------------------
+
+*    :html:`<iframe>` to `cdn.example.com`, `*.youtube.com` should be allowed
+*    :html:`<img>` sources to `cdn.example.com` and `*.instagram.com` should be allowed
+*    :html:`<script>` sources to `cdn.example.com` and `*.youtube.com` and `*.google.com`
+     should be allowed
+
+Rules for the TYPO3 backend
+---------------------------
+
+Normal TYPO3 backend rules need to be applied, so we only want to add some
+rules for custom backend modules:
+
+*    :html:`<iframe>` to `cdn.example.com` should be allowed
+*    :html:`<img>` sources to `cdn.example.com` should be allowed
+*    :html:`<script>` sources to `cdn.example.com` should be allowed
+
+Resulting configuration example:
+--------------------------------
+
+And this is how you would do that with a CSP YAML configuration file, one per site:
+
+..  literalinclude:: _csp_example_com.yaml
+    :language: yaml
+    :caption: config/sites/example-com/csp.yaml | typo3conf/sites/example-com/csp.yaml
+
+..  literalinclude:: _csp_example_org.yaml
+    :language: yaml
+    :caption: config/sites/example-org/csp.yaml | typo3conf/sites/example-org/csp.yaml
+
+..  literalinclude:: _ContentSecurityPolicies_example.php
+    :language: yaml
+    :caption: EXT:my_extension/Configuration/ContentSecurityPolicies.php
+
+This is really just a simple demo, that has room for improvements. For example,
+the allowed list of `src` values to any directive could actually be set through their common
```

Comment:
```suggestion
the allowed list of `*-src` values to any directive could actually be set through their common
```

This makes it clearer. First I thought about `<tag src="">`. Maybe add examples like font-src and img-src to make it clearer.